js/ui/drop_down_button.js

```diff
@@ -520,8 +523,19 @@ const DropDownButton = Widget.inherit({
 
     _updateItemCollection(optionName) {
         this._setWidgetOption('_list', [optionName]);
-        this._setListOption('selectedItemKeys', []);
-        this._loadSelectedItem().done(this._updateActionButton.bind(this));
+        this.getDataSource().store().byKey(this.option('selectedItemKey')).done(selectedItem => {
```

Comment:
This `byKey` call raises the `Data item cannot be found` error when there is no necessary item in the new item. Is it okay?